tasks/managed/apply-mapping/apply-mapping.yaml

```diff
@@ -353,3 +391,29 @@ spec:
               "${SNAPSHOT_SPEC_FILE}" > /tmp/temp && mv /tmp/temp "${SNAPSHOT_SPEC_FILE}"
             fi
         done
+    - name: create-trusted-artifact
```

Comment:
The first step says we will skip trusted-artifacts steps if the param is empty, but I don't see any when condition on this step. Is that intended?

Reply:
seems you cannot guard Steps from execution ... only Tasks in a Pipeline

the mechanism for skipping involves creating a `skip file` in the extract or create directory.